### ISSUE TITLE
instance attribute .maximize_window shadows method BaseCase.maximize_window()

### ISSUE THREAD

**Issue by plumiron** [seleniumbase/SeleniumBase#363]
Hi,
I have some code as following to maximize the browser window:
```python
from seleniumbase import config as sb_config
from seleniumbase import BaseCase

class VerifyMaximizeWindow(BaseCase):
    def test_maximize_window(self):
        url = "https://10.109.109.181"
        self.open(url)
        self.maximize_window()
```
However it complains that:
```python
================================================= test session starts =================================================
platform win32 -- Python 3.7.4, pytest-5.1.1, py-1.8.0, pluggy-0.12.0
rootdir: C:\pyags\pyags_test
plugins: cov-2.7.1, forked-1.0.2, html-1.22.0, metadata-1.8.0, ordering-0.6, rerunfailures-7.0, timeout-1.3.3, xdist-1.2
9.0, seleniumbase-1.30.0
collected 1 item

testcase/test_maximize_window.py::VerifyMaximizeWindow::test_maximize_window
DevTools listening on ws://127.0.0.1:55285/devtools/browser/d8e18d4e-3efa-4f30-a9ba-3cf1ae466a78
FAILED

====================================================== FAILURES =======================================================
______________________________________ VerifyMaximizeWindow.test_maximize_window ______________________________________

self = <test_maximize_window.VerifyMaximizeWindow testMethod=test_maximize_window>

    def test_maximize_window(self):
        url = "https://10.109.109.181"
        self.open(url)
>       self.maximize_window()
E       TypeError: 'bool' object is not callable

testcase\test_maximize_window.py:9: TypeError
================================================== 1 failed in 6.18s ==================================================
```

Looks like SeleniumBase set self.maximize_window in BaseCase.setUp(), which shadows the method with the same name.
```python
            self.extension_dir = sb_config.extension_dir
            self.maximize_window = sb_config.maximize_window
            self.save_screenshot_after_test = sb_config.save_screenshot
```

**Comment by mdmintz** [comment]
Hi @plumiron Thanks for catching that! I have fixed this issue in seleniumbase 1.31.1 with https://github.com/seleniumbase/SeleniumBase/pull/365

**Comment by mdmintz** [comment]
Since this issue is now fixed, I’m closing this ticket.
